### PR TITLE
fix(typos): various minor unrelated fixes 

### PR DESCRIPTION
**Pull request by tkrugg**
Theses are fixes I've made on the go while working on react-native compat.
They are small enough that I grouped them in a single PR.

### CODE REVIEW

File: lib/init.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import { isUndefined, isString } from "./utils/index";
-import { isNumber } from "util";
```
Comment (samouss) [inline lib/init.ts]: Which means that we have `util` in the current build of the lib?

Comment (Haroenv) [reply lib/init.ts]: let's add bundlesize to avoid this in the future!

Comment (tkrugg) [reply lib/init.ts]: 🤦‍♂️ my mistake.
No it's not in the bundle. rollup doesn't include node builtins unless you tell it to 
https://www.npmjs.com/package/rollup-plugin-node-builtins

Comment (tkrugg) [reply lib/init.ts]: Actually you're right. It was included in the bundle 😱 +3kB

**Pull request merged**